### PR TITLE
Consolidates sending first email topic

### PR DESCRIPTION
**Pull request by scribblingfox**
Couple questions in the PR body that we should answer before I open this for real. 

One, i'll ask here: Do we like the placement of "Sending your first email" in the side nav, or should it be closer to the top of the section, by the quickstart? It seems like there's a lot of optional stuff between the Quickstart and this topic, and some new users might just want to start firing away emails.

### CODE REVIEW

File: pages/getting-started/sending-your-first-email.mdx [hunk 1]
```diff
@@ -74,31 +205,43 @@ Hello friend!
...
 
-### Updating your email's issue number
+### Update your email's issue number
 
-Buttondown tries to be clever with determining what issue number your newsletter is on. Sometimes it is either _too_ clever or not nearly clever enough, though, and you end up in a situation where your seventh issue is showing up as your eighth or vice-versa. This is particularly pernicious as a problem if you're bringing in archives from another system.
+Buttondown tries determine what issue number your newsletter is on. For example, your first email will be labeled number 1. But it doesn't always get this right. This is particularly a problem for people who have imported the archives of an existing newsletter from another system. If you need to, you can manually edit your issue numbers in your Archive page. 
 
-You can manually edit your issue numbers by going to the **Edit Email** screen in your Archive page. The **ID** field corresponds with the issue number:
+<!-- TKTK This is another thing that wants a procedure, but I can't find the UI. -->
```
Comment (scribblingfox) [inline pages/getting-started/sending-your-first-email.mdx:230]: @jmduke I couldn't get this modal to appear anywhere in the emails page, or by searching for it. How do I find it?

File: pages/getting-started/sending-your-first-email.mdx [hunk 2]
```diff
@@ -16,54 +16,185 @@ export default ({ children }) => <Layout meta={meta}>{children}</Layout>;
...
+
+You don't have to send an email as soon as you write it. You can schedule it to send later, at a specific date or time. You can do this by clicking the “Scheduling” button at the bottom of your draft.
+
+To learn more, read [Scheduling an Email](/getting-started/scheduling-an-email).
+
+## Images
+
+Buttondown uses Amazon S3 (a common file hosting platform) to safely and securely store your images. Images you upload to Buttondown last indefinitely by default. This means that even if you were to delete the email in which images are referenced, the images themselves still exist on Buttondown's servers.
+
+<!-- TKTK What file formats are supported? Also should add real procedure here. -->
```
Comment (scribblingfox) [inline pages/getting-started/sending-your-first-email.mdx:149]: @jmduke What file formats are supported?

File: pages/getting-started/sending-your-first-email.mdx [hunk 3]
```diff
@@ -16,54 +16,185 @@ export default ({ children }) => <Layout meta={meta}>{children}</Layout>;
...
-There are many ways to apply a tag to a subscriber, once you've created them:
+There are a few different ways to apply a tag to a subscriber:
 
+- Tagging subscribers manually
 - Customizing your [subscription form](https://blog.buttondown.email/2019/07/17/customizing-your-subscription-form-to-tag-new-subscribers)
-- Manually tagging them
-- Setting them via the [API](https://docs.buttondown.email/api-reference/introduction)
+- Setting tags with the [API](/api-reference/introduction)
+
+<!-- TKTK Should document how to manually tag subscribers, at least -->
```
Comment (scribblingfox) [inline pages/getting-started/sending-your-first-email.mdx:177]: @jmduke I couldn't get this modal to appear anywhere in the emails page, or by searching for it. How do I find it?

**Comment by scribblingfox** [comment]
@jmduke Opening this for review so I can get your input on my questions. Not actually ready to merge yet!

**Comment by jmduke** [comment]
> One, i'll ask here: Do we like the placement of "Sending your first email" in the side nav, or should it be closer to the top of the section, by the quickstart? It seems like there's a lot of optional stuff between the Quickstart and this topic, and some new users might just want to start firing away emails.

This is a great question that I should probably have a stronger answer to. I think right now the golden ideal path for activation is something like:

1. Registration
2. Basic setup (name, description, etc)
3. Importing old data
4. Sending your first email
5. Doing fancy stuff

So let's push it up in the nav up to reflect that better.

File: pages/getting-started/sending-your-first-email.mdx (hunk 1 above)
Comment (jmduke) [reply pages/getting-started/sending-your-first-email.mdx:230]: That doc is straight up wrong and has been for what is, in retrospect, a disturbing amount of time. You change it now by going to "edit" (which brings you back to the main screen where you write the email) and then clicking on "Web archive".

File: pages/getting-started/sending-your-first-email.mdx (hunk 2 above)
Comment (jmduke) [reply pages/getting-started/sending-your-first-email.mdx:149]: Everything: ['blp', 'bmp', 'dib', 'bufr', 'cur', 'pcx', 'dcx', 'dds', 'ps', 'eps', 'fit',
 'fits', 'fli', 'flc', 'ftc', 'ftu', 'gbr', 'gif', 'grib', 'h5', 'hdf', 'png',
 'apng', 'jp2', 'j2k', 'jpc', 'jpf', 'jpx', 'j2c', 'icns', 'ico', 'im', 'iim',
 'tif', 'tiff', 'jfif', 'jpe', 'jpg', 'jpeg', 'mpg', 'mpeg', 'mpo', 'msp',
 'palm', 'pcd', 'pdf', 'pxr', 'pbm', 'pgm', 'ppm', 'pnm', 'psd', 'bw', 'rgb',
 'rgba', 'sgi', 'ras', 'tga', 'icb', 'vda', 'vst', 'webp', 'wmf', 'emf', 'xbm',
 'xpm']

File: pages/getting-started/sending-your-first-email.mdx (hunk 3 above)
Comment (jmduke) [reply pages/getting-started/sending-your-first-email.mdx:177]: By going to a subscriber's detail page!

image

**Comment by scribblingfox** [comment]
@jmduke Cool, I've moved the first email topic up in the sidenav and edited in your other comments. Thanks for the feedback!

**Review by jmduke — APPROVED** [review]

**Pull request merged**